geoips/plugins/modules/sector_metadata_generators/tc_sector_file_parser.py

```diff
@@ -24,8 +23,12 @@
 
 LOG = logging.getLogger(__name__)
 
+interface = "sector_metadata_parser"
+family = "tc"
```

Comment:
Are we good with `family = "tc"` or would it be better to do `family = "tropical_cyclone"`? I've been using `tc`, too.

Reply:
We could actually do tropical_cyclone ... let's change it in a different PR though